internal/reconcilers/pipeline_dependencies.go

```diff
@@ -82,23 +90,44 @@ func (r *PipelineDependenciesReconciler) Reconcile(ctx context.Context, pipeline
 
 	for _, task := range tasks {
 		if !task.IsDir() {
-			if err := r.applyManifests(ctx, filepath.Join(taskManifestsPath, task.Name()), pipeline, new(tekton.Task)); err != nil {
+			if err := r.applyManifests(ctx, filepath.Join(taskManifestsPath, task.Name()), pipeline, new(tekton.Task), false); err != nil {
 				return true, err
 			}
 		}
 	}
 
+	if err := r.applyManifests(ctx, filepath.Join(gitPath, baseManifestsPath, sccYml), pipeline, new(securityv1.SecurityContextConstraints), true); err != nil {
+		return true, err
+	}
+
+	if err := r.applyManifests(ctx, filepath.Join(gitPath, baseManifestsPath, clusterRoleYml), pipeline, new(rbacv1.ClusterRole), true); err != nil {
+		return true, err
+	}
+
+	// apply the cluster role binding with modifications below
+	tmpFile, err := r.modifyAndSaveTempClusterRoleBinding(ctx, filepath.Join(gitPath, baseManifestsPath, clusterRoleBindingYml), pipeline, new(rbacv1.ClusterRoleBinding))
+	if err != nil {
+		return true, err
+
+	}
+	//todo-adam is this necessary? isn't temp cleaned up or no? probably better safe then sorry?
```

Comment:
Leftover comment

Reply:
The comment is valid, I meant to ask others if this is needed or not. :)

What are your thoughts? do we usually cleanup on files put in a `tmp` dir?

Reply:
Can't hurt to clean up. I'd say it's fine.